src/api/string.js

```diff
@@ -44,92 +51,184 @@ async function asyncReplace(source, pattern, asyncReplacer) {
     return strings.join('');
 }
 
-function StartsWith(str, sub_str) {
-    return ('' + str).indexOf(sub_str) === 0;
+/**
+ * @param {string} str
+ * @param {string} sub_str
+ * @return {boolean}
+ * @deprecated Use `str.startsWith(sub_str)`
+ */
+function startsWith(str, sub_str) {
+    return String(str).indexOf(sub_str) === 0;
```

Comment:
`String(str)` is faster than `('' + str)`, especially when `str` is already a `string`.